src/lib/parse_pdbqt.cpp

```diff
@@ -620,7 +617,14 @@ struct pdbqt_initializer {
 model parse_ligand_pdbqt_from_file(const std::string& name, atom_type::t atype) { // can throw parse_error
     non_rigid_parsed nrp;
     context c;
-    parse_pdbqt_ligand(make_path(name), nrp, c);
+
+    try {
+        parse_pdbqt_ligand(make_path(name), nrp, c);
+    }
+    catch(pdbqt_parse_error& e) {
+        std::cerr << e.what();
```

Comment:
It looks like none of the error strings when `pdbqt_parse_error` is thrown contain a final newline `\n` - so it should probably be added here.

Reply:
Also, for consistency with the other error outputs maybe preface the errors with "\n\nParse error: "